tidb-binlog-deployment-topology.md

```diff
@@ -22,24 +22,33 @@ TiDB Binlog is the widely used component for replicating incremental data. It pr
 
 ### Topology templates
 
-- [The simple template for the TiDB Binlog topology](https://github.com/pingcap/docs/blob/master/config-templates/simple-tidb-binlog.yaml)
+- [The simple template for the TiDB Binlog topology (the downstream type is `MySQL`)](https://github.com/pingcap/docs/blob/master/config-templates/simple-tidb-binlog.yaml)
+- [The simple template for the TiDB Binlog topology (the downstream type is `file`)](https://github.com/pingcap/docs/blob/master/config-templates/simple-file-binlog.yaml)
 - [The complex template for the TiDB Binlog topology](https://github.com/pingcap/docs/blob/master/config-templates/complex-tidb-binlog.yaml)
 
 ### Key parameters
 
 The key parameters in the topology configuration templates are as follows:
 
-- `binlog.enable: true`
+- `server_configs.tidb.binlog.enable: true`
 
     - Enables the binlog service.
     - Default value: `false`.
 
-- `binlog.ignore-error: true`
+- `server_configs.tidb.binlog.ignore-error: true`
 
     - It is recommended to enable this configuration in high availability scenarios.
     - If set to `true`, when an error occurs, TiDB stops writing data into binlog, and adds `1` to the value of the `tidb_server_critical_error_total` monitoring metric.
     - If set to `false`, when TiDB fails to write data into binlog, the whole TiDB service is stopped.
 
+- `drainer_servers.config.syncer.db-type`
+
+    The downstream type of binlog. Currently, `mysql`, `tidb`, `kafka` and `file` are supported.
+
+- `drainer_servers.config.syncer.to`
+
+    The downstream configuration of binlog. Depending on different `db-type`s, this parameter can configure the connection parameters of the downstream database, the connection parameters of Kafka, and the file save path. For details, refer to [TiDB Binlog Configuration File](/tidb-binlog/tidb-binlog-configuration-file.md#syncerto).
```

Comment:
@lance6716 Does the `binlog` here mean TiDB Binlog?
```suggestion
    The downstream configuration of binlog. Depending on different `db-type`s, this parameter can configure the connection parameters of the downstream database, the connection parameters of Kafka, and the file save path. For details, refer to [TiDB Binlog Configuration File](/tidb-binlog/tidb-binlog-configuration-file.md#syncerto).
```

Reply:
Yes 😂

Reply:
@Joyinqin Would you help update the [Chinese version](https://github.com/pingcap/docs-cn/pull/5049/files#diff-2f168035c74d14f399fcf70b83f2475de46b737c1cc425ab936b0871810a3dd4R41-R48) (binlog --> TiDB Binlog)? Thanks!

Reply:
Update in https://github.com/pingcap/docs-cn/pull/5120.